trigger/handler.py

```diff
@@ -50,13 +52,26 @@ def process_individual_payload(individual_payload):
                 # handle things are coming through for the first time (i.e. they haven't failed before)
                 for s3_record in s3_record_list:
                     s3_object_size = s3_record['s3']['object']['size']
+                    print(f"object size {s3_object_size} limit {s3_object_size_limit}")
                     if int(s3_object_size) < s3_object_size_limit:
                         raw_payload = {'Record': s3_record}
                         payload = json.dumps(raw_payload)
                         process_individual_payload(payload)
-                    # ignore giant s3 files for now
+                    # put giant s3 files into the chopping queue
                     else:
-                        logger.info(f'Omitted {s3_record} because it exceeded the set file size limit for loading.')
+                        print("sending to chopper")
```

Comment:
Might not need the print statement anymore, though I do like it.

Reply:
will remove